src/coreclr-debug/util.ts

```diff
@@ -127,11 +127,11 @@ export function getTargetArchitecture(platformInfo: PlatformInformation, launchJ
     }
 
     // Otherwise, look at the runtime ID.
-    if (dotnetInfo.RuntimeId.includes('arm64'))
+    if (dotnetInfo.RuntimeId?.includes('arm64'))
```

Comment:
If we are starting to debug but only have a runtime-only dotnet, should we have an error message to install an SDK runtime?

@gregg-miskelly

Reply:
I don't know of any reason why we couldn't debug without an SDK.

Reply:
What about:
```
if (!dotnetInfo.RuntimeId) {
   throw new Error(`Unable to determine RuntimeId. Please set 'targetArchitecture' in your launch.json configuration.`)
} else {
   if (dotnetInfo.RuntimeId.Contains....)

   ....

   throw new Error...
}
```